lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1463,9 +1466,10 @@ ParseResult ClassOp::parse(OpAsmParser &parser, OperationState &result) {
   SmallVector<Attribute, 4> portAnnotations;
   SmallVector<Attribute, 4> portSyms;
```

Comment:
It would be nice if we don't have to declare `portSyms`  but the current implementation makes sense (maybe we can use `std::optinal<SmallVectorImpl<Attribute>&>` but would be messier).

Reply:
Agreed! With all the in-flight module/port reworkings I vote leave this as-is and revisit in a bit.  But yes this could be improved!